nad/epsg

```diff
@@ -1,3 +1,4 @@
+<metadata> +version=9.0.0 +origin=EPSG +lastupdate=2017-01-10
```

Comment:
This change will be lost at the next upgrade of EPSG database. Perhaps https://github.com/OSGeo/gdal/blob/trunk/gdal/swig/python/scripts/epsg_tr.py should be altered to add it. But it is called several times as mentionned towards the end of https://trac.osgeo.org/geotiff/browser/trunk/libgeotiff/csv/README . Or perhaps just this README file should be modified to mentionn adding this header line